docs/UserGuide.md

```diff
@@ -44,78 +41,102 @@ a place to start or a veteran needing a quick reference, this guide is here to a
 
-| Icon               | Meaning                                                       |
-|--------------------|---------------------------------------------------------------|
-|:information_source:| This icon indicates important information to be taken note of |
-|:bulb:               | This icon indicates useful tips for the users                |
+### Overview of InternBuddy
+InternBuddy is designed specifically for students who are searching for internships. With InternBuddy, there is no need for you to juggle between multiple apps just to manage your multiple internship applications. With efficiency as its focus, InternBuddy values your **Time** and **Experience**. Let InterBuddy help you to manage your application details so you can focus acing your assessments and interviews!
+### Structure of InternBuddy
+With internships and job seeking as our main focus, we are going over 3 main entries of InternBuddy:
+- **Company**: A company will represent a  company that you are applying for, or will be applying for.
+- **Person**: A person will represent a contact person. In can be used for general purpose of storing a contact person with a company related to it. The contact person may have a specific role such as the HR of a company.
+- **Event**: An event will represent an event of a company. An event can be an online assessment, an interview, a job fair, etc.
 
---------------------------------------------------------------------------------------------------------------------
-## Overview of InternBuddy
-
-InternBuddy is designed specifically for students who are searching for internships. With InternBuddy, there is no need 
-for you to juggle between multiple apps just to manage your multiple internship applications. With efficiency as 
-its focus, InternBuddy values your **Time** and **Experience**. Let InterBuddy help you to manage your application details
-so you can focus acing your assessments and interviews!
+And with that, InternBuddy stores 3 lists for 3 different types of entries:
+- **[Companies](#company-list)**
+- **[Persons](#person-list)**
+- **[Events](#event-list)**
 
-InternBuddy stores 3 lists for 3 different types of entries: **Companies**, **Persons**, and **Events**. 
-The app will only display one of these lists at any time, as shown in the screenshot below.
+The app will only display one of these lists at any time, as shown in the screenshot below. Each of these 3 different types have different attributes attached to them. Let's dive in to understand what each type can store.
 
-### Company List
+#### Company List
 ![Company List](images/UserGuide/Ui-2.png)
 
-### Person List
-![Person List](images/UserGuide/Ui-3.png)
-
-### Event List
-![Event List](images/UserGuide/Ui-4.png)
-
-Each of these 3 different types have different attributes attached to them. Let's dive in to understand what each type
-can store.
-
 A Company entry has:
```

Comment:
Perhaps 'Company' can be bolded. Similarly for 'Person' and 'Event' because I think bolding entries could be helpful.

Reply:
Done!